ansible_collections/arista/avd/examples/campus-fabric/README.md

```diff
@@ -0,0 +1,390 @@
+
+    Variables applied under the node key type (spine/leaf) defaults section are inherited to nodes under each type. These variables may be overwritten under the node itself.
+
+    The spine interface used by a particular leaf is defined from the leaf's perspective with a variable called `uplink_switch_interfaces`. For example, LEAF2A has a unique variable `uplink_switch_interfaces: [Ethernet49/1, Ethernet49/1]` defined. This means that LEAF2A is connected to SPINE1's Ethernet49/1 and SPINE2's Ethernet49/1, respectively.
+
+    ``` yaml
+    --8<--
+    examples/campus-fabric/group_vars/DC1_FABRIC.yml
+    --8<--
+    ```
+
+=== "DC1_SPINES"
+    In an L2LS design, there are two types of spine nodes: `spine` and `l3spine`. In AVD. the node type defines the functionality and the EOS CLI configuration to be generated. For an L2LS design, we will use node type: spine. Later, we will add routing to the Spines by changing the node type to l3spine.
+
+    ``` yaml
+    --8<--
+    examples/campus-fabric/group_vars/DC1_SPINES.yml
+    --8<--
+    ```
+
+=== "DC1_LEAFS"
+    In an L2LS design, we have one type of leaf node: `leaf`.
+
+    ``` yaml
+    --8<--
+    examples/campus-fabric/group_vars/DC1_LEAFS.yml
+    --8<--
+    ```
+
+=== "DC1_NETWORK_SERVICES"
+    You add VLANs and SVIs to the Fabric by updating the **group_vars/DC1_NETWORK_SERVICES.yml**. Each VLAN will be given a name and a list of tags. The tags filter the VLAN to specific Leaf Pairs. These variables are applied to spine and leaf nodes since they are a part of this group.
+
+    ``` yaml
+    --8<--
+    examples/campus-fabric/group_vars/DC1_NETWORK_SERVICES.yml
+    --8<--
+    ```
+
+=== "DC1_NETWORK_PORTS"
+    Our fabric would not be complete without connecting some devices to it. We define port profiles and network port ranges in **group_vars/DC1_NETWORKS_PORTS.yml**. A single port_profile may be used across a number of switches and port ranges. In our example we create a port_profile called `PP-DOT1X` to define generic 802.1x (NAC) settings we wish to apply to a range of ports. The network_ports data model is where we define which switches and ports to apply the port profile. This data model allows a single regex statement to define a list of switches. In addition, the variable `switch_ports` expands into a range of ports. Details of the range_expand filter can be viewed [here](https://avd.sh/en/stable/plugins/index.html#range_expand-filter). These variables are applied to spine and leaf nodes since they are a part of this inventory group.
```

Comment:
```suggestion
Our fabric would not be complete without connecting some devices to it. Therefore, we define port profiles and network port ranges in the  **group_vars/DC1_NETWORKS_PORTS.yml** file. A single port_profile may be used across several switches and port ranges. In our example, we create a port profile called `PP-DOT1X` to define generic 802.1x (NAC) settings we wish to apply to a range of ports. The `network_ports` data model is where we define which switches and ports to apply the port profile. This data model allows a single regex statement to define a list of switches. In addition, the variable `switch_ports` expands into a range of ports. You can view more details of the `range_expand` filter [here](https://avd.sh/en/stable/plugins/index.html#range_expand-filter). These variables are applied to spine and leaf nodes since they are a part of this inventory group.
```

Reply:
done